tests/provisioning-manifest-build.yml

```diff
@@ -1 +1,2 @@
 - sleep: 10
+- include: 'mvn:org.jahia.packages/jahia-additional-modules/${const:org.jahia.api.Constants.JAHIA_PROJECT_VERSION}/yaml/provisioning'
```

Comment:
Why running the provisioning for "jahia-additional-modules" ?

Reply:
Test is creating a site using a template that is in a module